nvm.sh

```diff
@@ -2824,7 +2829,7 @@ nvm_auto() {
       nvm install >/dev/null
     fi
   elif [ "_$NVM_MODE" = '_use' ]; then
-   VERSION="$(nvm_resolve_local_alias default 2>/dev/null || nvm_echo)"
```

Comment:
Oh right! You are correct. I must have made something  weird within vi. Want me to fix this?